koku/api/tags/queries.py

```diff
@@ -80,6 +70,11 @@ def __init__(self, parameters):
             parameters    (QueryParameters): parameter object for query
 
         """
+        if parameters.parameters.get("filter", {}).get("time_scope_value") == "-10":
+            parameters.parameters["filter"]["time_scope_value"] = "-1"
+            parameters.parameters["filter"]["time_scope_units"] = "month"
+            parameters.parameters["filter"]["resolution"] = "monthly"
```

Comment:
`get_filter()` can be used to simplify this.

Reply:
These lines have been removed.